requirements.txt

```
@@ -7,3 +7,4 @@ requests>=2.24.0
 xarray>=0.19
 zarr>=2.5
 pydantic>=1.8.2
+git+git://github.com/NCAR/xcollection.git
```

I think it needs to be a valid HTTP url

```suggestion
git+https://github.com/NCAR/xcollection
```